capture.py

```diff
@@ -352,16 +352,30 @@ def apply_view(panel, **options):
         cmds.setAttr(attr, value)
 
 
-def parse_active_view():
-    """Parse the current settings from the active view"""
+def parse_active_panel():
+    """Parse the active modelPanel.
+
+    Raises
+        RuntimeError: When no active modelPanel an error is raised.
+
+    Returns:
+        str: Name of modelPanel
+
+    """
 
     panel = cmds.getPanel(wf=True)
```

Comment:
This is what I wanted to nitpick on, even if it didn't occur in this PR; would it be possible to expand on the argument, from wf to what I'm assuming is withFocus? Got stuck reading here for a bit.